test/system/mentor/registration_test.rb

```diff
@@ -25,4 +25,36 @@ class MentorRegistrationTest < ApplicationSystemTestCase
     assert user.is_mentor?
     assert_equal [ruby], user.mentored_tracks
   end
+
+  test "logging in mentor flow works" do
+    password = "foobar"
+    user = create :user, password: password
+    user.confirm
```

Comment:
Not `confirm!` (with bang)?

Reply:
No, weirdly. Feels like it should be but Devise defines it without.

Reply:
😕